data/online-2020-16.yaml

```diff
@@ -42,12 +42,17 @@ organizers:
   - name: Em Lazer-Walker
     email: me@lazerwalker.com
     twitterHandle: lazerwalker
+  - name: Alejandra
+    twitterHandle: AlxSavage
 thanks:
 sponsors:
   - name: Dashlane
     link: https://www.dashlane.com/
     media: https://i.imgur.com/LJ4sQ2B.png
 speakers:
+  - name: Vanessa Yuenyathei
```

Comment:
Vanessa Yuen is okay :)

Reply:
Thanks for the catch, I've updated it :)